redisearch/query.py

```diff
@@ -176,6 +182,14 @@ def add_filter(self, flt):
         self._filters.append(flt)
         return self
 
+    def sortby(self, sortfld):
```

Comment:
the API should be 

```
def sort_by(self, field, asc = True):
```

The internal class should not be intialized by the user.